spack/spack.yaml

```diff
@@ -0,0 +1,6 @@
+spack:
+  specs:
+    - py-pyprecice@develop
```

Comment:
Is there a risk that it will be misinterpreted because `@develop` might **not** point to the actual develop branch?

Reply:
I think for our testing this "misinterpretation" is exactly what we want: `dev-build` uses the current version of an arbitrary branch as a patch for `py-pyprecice@develop`. Then we install/use (the patched) `py-pyprecice@develop` in all our tests. Does this make sense? If yes: @ajaust feel free to mark this as resolved. If no: Let's try to get to a conclusion here. I should probably also document this somewhere...

Reply:
I removed the `spack.yaml` in e3371fd. I think this new approach is clearer (I hope it works as expected).